README.rst

```diff
@@ -12,9 +12,27 @@ on a given codebase.
 Getting Started - Development
 -----------------------------
 
+Running using Docker
+~~~~~~~~~~~~~~~~~~~~
+
+Set SECRET_KEY environment variable in ``docker-compose.yml`` to some random string.
```

Comment:
...and the variable is named SPARROW_SECRET_KEY in `docker-compose.yml`, those two could be consistent.